src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -118,8 +119,8 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
     this.infoTabService.defaultTestFilePathEditing$
       .pipe(takeUntil(this.ngUnsubscribe))
       .subscribe((editing) => (this.defaultTestFilePathEditing = editing));
+    this.entryType$.subscribe((entryType) => (this.displayTextForButton = '#' + entryType + '/' + this.workflow?.full_workflow_path));
```

Comment:
1. Needs `.pipe(takeUntil(this.ngUnsubscribe))` -- see Gary's comment about unsubscribing
2. Your string is not wrong and you can keep it that way, but for future reference, TypeScript has string interpolation which I personally prefer to inserting `+` signs and opening and closing quotes. You could write as `` `#${entryType}/${this.workflow?.full_workflow_path` ``. No need to change though

Reply:
let me know if I unsubscribed incorrectly

Reply:
> let me know if I unsubscribed incorrectly

I think you did -- look at lines 119-121 -- `pipe` then `subscribe`. And you need to chain them.